tools/val/val.cpp

```diff
@@ -63,8 +63,8 @@ int main(int argc, char** argv) {
     if ('-' == cur_arg[0]) {
       if (0 == strcmp(cur_arg, "--version")) {
         printf("%s\n", kBuildVersion);
-        printf("Target: SPIR-V %d.%d rev %d\n", SPV_SPIRV_VERSION_MAJOR,
-               SPV_SPIRV_VERSION_MINOR, SPV_SPIRV_VERSION_REVISION);
+        printf("Target: %s\n",
+               spvTargetEnvDescription(SPV_ENV_UNIVERSAL_1_0_4));
```

Comment:
Good point. And I assume the vulkan thing is not that relevant to as and dis. So I only added it for val.